pressbooks/pressbooks-book
# Adding specification in css rule to add priority for alignment margins

https://github.com/pressbooks/pressbooks-book/issues/704

### How to test
- Write/copy a large block of text, and also, import an image. Put the image on the same line as the text.
- Select an alignment (right or left). Notice that in the editor, there is a margin.
- Now, save and preview the chapter. You should see image have 1.5em of lef/right margin

_Please note content ID is the wrapper for all book web content: https://github.com/pressbooks/pressbooks-book/blob/pb-704-imgAlignmentIssue/header.php#L128_
